main.tf

```diff
@@ -4,6 +4,17 @@ provider "aws" {
   region                  = var.region
 }
 
+terraform {
+  backend "remote" {
+    hostname     = "app.terraform.io"
+    organization = "CovidZero"
+
+    workspaces {
+      name = "infrastructure"
```

Comment:
maybe we should use `prefix` instead of `name`, using prefix we can add a suffix like -prod or -staging, then it becomes easier to maintain.

Only one thing to consider, we have to set this suffix on GH Actions.